scripts/util/doc-builder.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+"use strict";
+
+const latestVersion = "v" + require('../../package.json').version;
+const path = require('path');
+const fs = require('fs');
+const yuidoc = require('./yuidoc');
+const NodeGit = require('nodegit');
+const util = require('util');
+const fsExtra = require('fs-extra');
+
+class DocBuilder {
```

Comment:
This class actually does some pretty complex things. Could you add comments to the functions describing what they do.